protostar/commands/test/starkware/test_execution_state.py

```diff
@@ -6,24 +6,29 @@
 from typing_extensions import Self
 
 from protostar.commands.test.test_context import TestContext
+from protostar.commands.test.test_output_recorder import OutputRecorder
 from protostar.starknet.execution_state import ExecutionState
 from protostar.starknet.forkable_starknet import ForkableStarknet
 from protostar.utils.starknet_compilation import StarknetCompiler
-from protostar.commands.test.test_output_recorder import OutputRecorder
 
 
 @dataclass
 class TestExecutionState(ExecutionState):
+    @dataclass
+    class Config:
+        max_fuzz_examples: int
```

Comment:
It is a value only related to `FuzzTestExecutionEnvironment` we shouldn't keep it in `TestExecutionState` config

Reply:
We can make a config that inherits from this config which includes max_fuzz_examples